DB.php

```diff
@@ -0,0 +1,11 @@
+hgjqw
+
+<q>wq
```

Comment:
i dont like this code 
```suggestion
<a> fuck u</a>
```